recipe/build.sh

```diff
@@ -10,12 +10,16 @@ fi
 
 if [[ -n "$tempest" && "$tempest" != "notempest" ]]; then
   export CONFIGURE_ARGS="--with-tempestremap=${PREFIX} --with-netcdf=${PREFIX} --with-netcdf-cxx=${PREFIX} ${CONFIGURE_ARGS}"
+else
+  export CONFIGURE_ARGS="--disable-blaslapack ${CONFIGURE_ARGS}"
+else
```

Comment:
I didn't fully understand the conversation about when to enable/disable BLAS and LAPACK.  @vijaysm, are you saying that E3SM will also want to run the no-tempest version of moab, and this will need to be with BLAS/LAPACK enabled?  Or is it as I have it here that we can disable BLAS/LAPACK when we're not using `tempest-remap`?